.github/workflows/docker-publish.yml

```
@@ -91,23 +92,35 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
 
-      - name: Build and export consumer test container to Docker
+      - name: Build and export nx consumer test container to Docker
         id: docker-build-consumer
         uses: docker/build-push-action@v3
         with:
           context: ./test-nx
           platforms: linux/amd64
           build-args: DOCKER_REPO=localhost:5000
           load: true
-          tags: ${{ env.TEST_CONSUMER_TAG }}
+          tags: ${{ env.TEST_NX_CONSUMER_TAG }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+      - name: Build and export app consumer test container to Docker
+        id: docker-build-consumer
+        uses: docker/build-push-action@v3
+        with:
+          context: ./test-nx
```

```suggestion
          context: ./test-app
```
I am assuming you want the test-app here